operators/pkg/controller/elasticsearch/initcontainer/prepare_fs.go

```diff
@@ -55,9 +59,18 @@ var (
 // - configuration changes
 // Modified directories and files are meant to be persisted for reuse in the actual ES container.
 // This container does not need to be privileged.
-func NewPrepareFSInitContainer(imageName string, linkedFiles LinkedFilesArray) (corev1.Container, error) {
+func NewPrepareFSInitContainer(
+	imageName string,
+	linkedFiles LinkedFilesArray,
+	transportCertificatesVolume volume.SecretVolume,
+) (corev1.Container, error) {
 	privileged := false
 	initContainerRunAsUser := defaultInitContainerRunAsUser
+
+	// we mount the certificates to a custom location to avoid the config moving to try to move files from the secret
```

Comment:
`to avoid the config moving to try to move files from the secret `: this is not super clear to me, can you maybe try to rephrase a little bit?

Reply:
rephrasing to:
```
	// we mount the certificates to a location outside of the default config directory because the prepare-fs script
	// will attempt to move all the files under the configuration directory to a different volume, and it should not
	// be attempting to move files from this secret volume mount (any attempt to do so will be logged as errors).
```